code/modules/vending/wardrobes.dm

```diff
@@ -123,7 +142,8 @@
 					/obj/item/clothing/gloves/fingerless = 2,
 					/obj/item/clothing/head/soft/black = 2,
 					/obj/item/clothing/mask/bandana/skull = 1)
-	premium = list(/obj/item/radio/headset/headset_rob = 2) //Cit change
+	contraband = list(/obj/item/storage/box/stockparts/basic = 1)
```

Comment:
Why is this contraband in the Roboticist Wardrobe but Premium in the Engineer's???? Still doesn't belong in a wardrobe.